apps/token/src/routes/governance/proposal/proposal-container.tsx

```diff
@@ -5,7 +5,7 @@ import { useTranslation } from 'react-i18next';
 import { useParams } from 'react-router-dom';
 
 import { SplashLoader } from '../../../components/splash-loader';
-import useFetch from '../../../hooks/use-fetch';
+import { useFetch } from '@vegaprotocol/react-helpers';
```

Comment:
Just want to verify that useFetch in token works in the same way as the version in react helpers and we can do a straight swap? If so 🙌

Reply:
Yes, the changes to the Explorer version were around passing optional parameters to refetch. I tested it manually on Token before commiting, all's well